### ISSUE TITLE
Bad output 'qc_report.qc_json_match'

### ISSUE THREAD

**Issue by vsoch** [ENCODE-DCC/chip-seq-pipeline2#16]
**Describe the bug**

This is a bug that pops up at the end of the pipeline run:

```
[2018-07-18 14:31:17,39] [info] PipelinesApiAsyncBackendJobExecutionActor [eca1fbc4chip.qc_report:NA:1]: Status change from Running to Success
[2018-07-18 14:31:19,33] [error] WorkflowManagerActor Workflow eca1fbc4-8be6-4401-90c4-8af43c017326 failed (during ExecutingWorkflowState): cromwell.backend.standard.StandardAsyncExecutionActor$$anon$2: Failed to evaluate job outputs:
Bad output 'qc_report.qc_json_match': java.nio.file.NoSuchFileException: gs://workflow-challenge/vanessasaurus/chip/eca1fbc4-8be6-4401-90c4-8af43c017326/call-qc_report/qc_json_match.txt
    at cromwell.backend.standard.StandardAsyncExecutionActor.$anonfun$handleExecutionSuccess$1(StandardAsyncExecutionActor.scala:824)
    at scala.util.Success.$anonfun$map$1(Try.scala:251)
    at scala.util.Success.map(Try.scala:209)
    at scala.concurrent.Future.$anonfun$map$1(Future.scala:288)
    at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
    at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
    at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
    at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
    at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:91)
    at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
    at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
    at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:91)
    at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)
    at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:43)
    at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
    at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
    at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
    at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

**OS/Platform and dependencies**
Google Cloud, run via a Docker Container, per instructions here -->

https://vsoch.github.io/wdl-pipelines/pipeline-chip-seq

**Attach logs**
The run, by way of a container, exited after finish so there aren't any logs. This should probably be advised to do as a bind to the container so they persist when it poops out.

**Comment by ximbao** [comment]
I'm having this issue running the pipeline in a local machine. Everything seems to be working but the QC step. Is there a way to resolve this? 

Thanks

**Comment by leepc12** [comment]
@ximbao: please run the following command on your working directory to make a tarball of all logs and attach it here.

```
$ find . -type f -name 'stdout' -or -name 'stderr' -or -name 'script' -or \
-name '*.qc' -or -name '*.txt' -or -name '*.log' -or -name '*.png' -or -name '*.pdf' \
| xargs tar -zcvf debug_ISSUE_16.tar.gz
```

**Comment by ximbao** [comment]
[debug_ISSUE_16.tar.gz](https://github.com/ENCODE-DCC/chip-seq-pipeline2/files/2479798/debug_ISSUE_16.tar.gz)

See attached file

**Comment by leepc12** [comment]
This will be fixed in the next release (commit baf87da1ef1b9f039b5be3a986fc15764a124a0b).
